GOVERNANCE.md

```
@@ -94,6 +97,46 @@ Depending on the reason for removal, a Maintainer may be converted to Emeritus
+## Members
+
+Members are active contributors who have shown a commitment to the project. They
+have privileges to review pull requests and are part of the
+`ovn-kubernetes/ovn-kubernetes-members` GitHub team, which makes them eligible
+for automatic PR review assignments. Members are not Maintainers, but they are
+expected to contribute to the project and collaborate with the team.
+
+### Becoming a Member
+
+To become a Member, you need to demonstrate the following:
+  * commitment to the project:
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)